BlockEntity/BEFruitpress.cs

```diff
@@ -230,17 +233,19 @@ private void onTick100msServer(float dt)
             double totalHours = Api.World.Calendar.TotalHours;
 
             double squeezeRel = mashStack.Attributes.GetDouble("squeezeRel", 1);
-            double litresToTransfer = Math.Min(juiceableLitresLeft, (totalHours - lastLiquidTransferTotalHours) * 50f);
+            double litresJuiceable = Math.Max(0, juiceableLitresLeft - ((juiceableLitresLeft + juiceableLitresTransfered) * screwPercent));
+
+            double litresToTransfer = Math.Min(litresJuiceable, (float)Math.Round((totalHours - lastLiquidTransferTotalHours) * 50f, 2));

```

Comment:
Rounding to two decimal points here removes the need for the rounding hack later.